src/UI/components/map/mapView/map-v2.tsx

```diff
@@ -139,6 +178,25 @@ export const MapWrapperV2 = () => {
         ></div>
       </Box>
       {selectedIds.length !== 0 && <DataDrawer />}
+      {areaModeOn ? (
+        <div
+          style={{
+            position: 'absolute',
+            right: 20,
+            top: 100,
+            zIndex: 10,
+            background: '#ebbd40',
+            boxShadow: '0 0 10px black',
+            paddingLeft: '20px',
+            paddingRight: '20px',
+            paddingTop: '5px',
+            paddingBottom: '5px',
+            color: 'black',
+          }}
+        >
+          <Typography>Area mode on</Typography>
+        </div>
```

Comment:
Out of interest, what happens when you download the image with area mode on? Does the image show the area?

Reply:
Cool, I wasn't sure if it'd show the `Area mode on` bit, but I guess that's not a layer.

Reply:
Ah yes - that's just a html div over the map.